src/main/java/uk/gov/digital/ho/proving/income/validator/AnnualisedAverageCalculator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.digital.ho.proving.income.validator;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Map;
+
+public class AnnualisedAverageCalculator {
```

Comment:
What does Annualised Average even mean?  Better to call this ProjectedAnnualIncomeCalculator?